include/RMGReaderMUSUN.hh

```diff
@@ -0,0 +1,43 @@
+#ifndef _RMG_GENERATOR_MUSUN_READER_HH_
+#define _RMG_GENERATOR_MUSUN_READER_HH_
+
+#include <fstream>
+#include <iostream>
+
+struct MUSUN_OUTPUT {
+    int nEvent;
+    int particleID;
+    float energy;
+    float x;
+    float y;
+    float z;
+    float theta;
+    float phi;
+};
+
+class RMGReaderMUSUN {
```

Comment:
Can you investigate if a MUSUN file can be interpreted as CSV, so we can use `G4CsvAnalysisReader` to parse it (see `RMGVertexFromFile`)?